src/main/java/com/juansecu/opentoonix/user/enums/ELoggingInError.java

```diff
@@ -3,7 +3,9 @@
 public enum ELoggingInError {
     USER_DISABLED("USER_DISABLED"),
     USER_NOT_EXISTS("USER_NOT_EXISTS"),
-    USER_WRONG_PASSWORD("USER_WRONG_PASSWORD");
```

Comment:
This error message must be present for every time a user tries to sign in with an wrong password at the moment.